### PR TITLE
Website deployment

### PR DESCRIPTION
**Pull request by calebjclark**
- Connected documentation search
- Fixed Assets page for downloading logos/etc
- Website now builds to build-dist for deployment
- `yarn deploy` deploys to Github Pages (secretagent.dev)

### CODE REVIEW

File: website/src/components/SearchForm.vue [hunk 1]
```diff
@@ -33,9 +33,9 @@
 
         import('docsearch.js').then(({ default: docsearch }) => {
           docsearch({
-            indexName: 'gridsome',
+            indexName: 'secretagent',
             inputSelector: `#${this.id}-input`,
-            apiKey: 'a7400a3a94b256c5283cb05efb860fc1',
+            apiKey: 'b975ccd9d9a93db63dd14f8d0baee5eb',
```
Comment (blakebyrnes) [inline website/src/components/SearchForm.vue:38]: Are we supposed to check these in?

File: website/src/pages/Why.vue [hunk 2]
```diff
@@ -33,7 +33,7 @@ BasicLayout.Why
         <a href="https://gs.statcounter.com/browser-version-market-share/desktop/worldwide/">top 3</a> most popular browsers are ready to plug-in.
 
 
-      h2 Writing Scraper Scripts is Still too Complicated
+      h2 Writing Scraper Scripts Is too Complicated
```
Comment (blakebyrnes) [inline website/src/pages/Why.vue:36]: Why capitalized? Shouldn't that be lower case?

File: website/src/components/SearchForm.vue (hunk 1 above)
Comment (calebjclark) [reply website/src/components/SearchForm.vue:38]: Yes, it's the public api key.

File: website/src/pages/Why.vue (hunk 2 above)
Comment (calebjclark) [reply website/src/pages/Why.vue]: Verbs should always be capitalized. And according to https://titlecaseconverter.com/, the word "too" should also be capitalized... just pushed an update.

**Pull request merged**